src/controllers/ioWalletController.ts

```diff
@@ -33,3 +50,97 @@ export const retrieveUserId = (
       )
     )
   );
+
+export default class IoWalletController {
+  constructor(private readonly ioWalletService: IoWalletService) {}
+
+  /**
+   * Get nonce
+   */
+  public readonly getNonce = (): Promise<
+    IResponseErrorInternal | IResponseSuccessJson<NonceDetailView>
+  > => this.ioWalletService.getNonce();
+
+  /**
+   * Create a Wallet Instance
+   */
+  public readonly createWalletInstance = (
+    req: express.Request
+  ): Promise<
+    | IResponseErrorInternal
+    | IResponseErrorGeneric
+    | IResponseErrorValidation
+    | IResponseSuccessJson<undefined>
```

Comment:
does this return 200? I'm asking since the OpenAPI reports 204

maybe `IResponseSuccessNoContent` is a better fit?